src/app/new/page.tsx

```
@@ -257,20 +419,132 @@ function NewChatPageInner() {
+        {mode === "pr" && (
+          <div className="mb-4 rounded-lg border border-border bg-card p-4">
+            <label
+              htmlFor="pr-url"
+              className="block text-sm font-medium text-foreground"
+            >
+              Pull request URL
+            </label>
+            <p className="mb-2 mt-0.5 text-[11px] text-muted-foreground">
+              Chorus shells out to{" "}
+              <code className="rounded bg-muted px-1">gh</code> on this machine
+              to fetch the PR's description, diff, and existing comments. You
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)